app/views/runs/_title.slim

```diff
@@ -45,6 +45,14 @@ h5
             span Claim
         - else
           button.btn.btn-outline-secondary disabled=true Sign In to Claim
+    - if can?(:create, RunLike)
+      button#like-button.btn.tip(
+        data={liked: current_user.likes?(run) ? '1' : '0'}
+        class=(current_user.likes?(run) ? 'btn-danger' : 'btn-outline-danger')
```

Comment:
500 when logged out on page load since `current_user` will be nil, then `likes?` is called on the nil object.